### PR TITLE
feat: Account details page hints

### LINKED ISSUE #52: Account details page hints
Display hints on account details page

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #52 

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

### CODE REVIEW

File: src/components/AccountNamesTableCondensed.vue [hunk 1]
```diff
@@ -55,6 +70,7 @@
 import AppLink from '@/components/AppLink'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { accountHints } from '~/utils/hints/accountHints'
```
Comment (lukeromanowicz) [inline src/components/AccountNamesTableCondensed.vue:73]: app-tooltip component is missing

File: src/components/AccountNamesTable.vue [hunk 2]
```diff
@@ -38,6 +53,8 @@
 import AppLink from '@/components/AppLink'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { accountHints } from '~/utils/hints/accountHints'
```
Comment (lukeromanowicz) [inline src/components/AccountNamesTable.vue:56]: I think we should stick to one type of aliases - @ in this case

here and in other components too

File: src/components/AccountTransactionsTableCondensed.vue [hunk 3]
```diff
@@ -50,6 +65,7 @@
 </template>
 
 <script setup>
+import { accountHints } from '../utils/hints/accountHints'
```
Comment (lukeromanowicz) [inline src/components/AccountTransactionsTableCondensed.vue:68]: app-tooltip component is missing

File: src/utils/hints/accountHints.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Current amount of AE owned by the account',
```
Comment (lukeromanowicz) [inline src/utils/hints/accountHints.js:2]: How about "Amount of AE currently owned by the account"

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
Most of the changes look really good. I suggested some minor improvements here and there

File: src/components/AccountNamesTable.vue (hunk 2 above)
Comment (janmichek) [reply src/components/AccountNamesTable.vue]: Right, fixed

File: src/components/AccountNamesTableCondensed.vue (hunk 1 above)
Comment (janmichek) [reply src/components/AccountNamesTableCondensed.vue]: Sometimes I forgot them, sometimes I didn't import it intentionally. I wanted to see how the autoimport works. Seems like for months there has been no issue with it and no one noticed. It's the thing I wanted to discuss on last conference, but there was very little time to cover all topics. I would like to give it a try. For the same reason we use magical auto router. I see imports as a technical boilerplate given the framework we are using. Could we discuss it here? https://github.com/aeternity/aescan/issues/112

File: src/utils/hints/accountHints.js (hunk 4 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js]: Nice suggestion, accepted

File: src/components/AccountNamesTableCondensed.vue (hunk 1 above)
Comment (lukeromanowicz) [reply src/components/AccountNamesTableCondensed.vue]: Yes, currently, the autoimport of components, pages, and stores is activated and we don't need to use it. I'm saying it only because we agreed to still import files until #112 is approved by everybody and resolved.

File: src/utils/hints/accountHints.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:2]: I would remove "currently"

File: src/utils/hints/accountHints.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
+  hash: 'Transaction hash of transaction sent by account.',
+  time: 'Estimated date, time, and block height when the transaction has been sent from the account.',
+  type: 'Type of the transaction sent by account.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:10]: It's not always sent, it could also be a transaction where the account is the receiver.

File: src/utils/hints/accountHints.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
+  hash: 'Transaction hash of transaction sent by account.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:8]: It's not always sent, it could also be a transaction where the account is the receiver.

File: src/utils/hints/accountHints.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  time: 'Estimated date, time, and block height when the transaction has been sent from the account.',
+  type: 'Type of the transaction sent by account.',
+  name: 'Name owned by this account.',
+  expires: 'Estimated date, time, and block height when the name owned by the account expires.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:12]: I might be wrong but If I read this it seems that the block height is also estimated. I would maybe change it to:
"Block height and estimated date when..."

File: src/utils/hints/accountHints.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
+  hash: 'Transaction hash of transaction sent by account.',
+  time: 'Estimated date, time, and block height when the transaction has been sent from the account.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:9]: I might be wrong but If I read this it seems that the block height is also estimated. I would maybe change it to:
"Block height and estimated date when..."

File: src/utils/hints/accountHints.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  type: 'Type of the transaction sent by account.',
+  name: 'Name owned by this account.',
+  expires: 'Estimated date, time, and block height when the name owned by the account expires.',
+  pointsTo: 'Account that the name owned by the account points to.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:13]: I would rephrase: "The account that points to the name owned by the account".

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I added some suggestions.

File: src/utils/hints/accountHints.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/accountHints.js:4]: fixed

File: src/utils/hints/accountHints.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/accountHints.js:3]: removed

File: src/utils/hints/accountHints.js [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/accountHints.js:7]: fixed

File: src/utils/hints/accountHints.js (hunk 6 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js:10]: fixed

File: src/utils/hints/accountHints.js (hunk 7 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js:8]: fixed

File: src/utils/hints/accountHints.js (hunk 8 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js:12]: fixed

File: src/utils/hints/accountHints.js (hunk 9 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js:9]: fixed

File: src/utils/hints/accountHints.js (hunk 10 above)
Comment (janmichek) [reply src/utils/hints/accountHints.js:13]: Fixed

File: src/components/AccountNamesTableCondensed.vue (hunk 1 above)
Comment (janmichek) [reply src/components/AccountNamesTableCondensed.vue]: For now I added the imports and initiated discussion

**Review by CharisSiarampalis — APPROVED** [review]

File: src/utils/hints/accountHints.js [hunk 14]
```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE owned by the account.',
+  value: 'Fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions where the account is involved.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:4]: Maybe `where the account was involved.` is better?

File: src/utils/hints/accountHints.js [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Node API link of the account.',
+  hash: 'Transaction hash of transaction where the account is involved.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:8]: `Transaction hash of the transaction where the account was involved.` sounds better to me.

File: src/utils/hints/accountHints.js [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+  time: 'Block height and estimated date and time when the transaction has been sent from the account.',
+  type: 'Type of the transaction where the account is involved.',
```
Comment (michele-franchi) [inline src/utils/hints/accountHints.js:10]: `Type of the transaction where the account was involved.`

**Review by michele-franchi — APPROVED** [review]
Looks good but I added some more suggestions.

**Comment by janmichek** [comment]
Thanks, all accepted @michele-franchi
